README.md

```diff
@@ -351,6 +348,9 @@ and reports back results to the queue.
   Usage:
     generic-worker run                      [--config         CONFIG-FILE]
                                             [--configure-for-aws | --configure-for-gcp]
+    generic-worker install service          [--service-name SERVICE-NAME]
```

Comment:
Can we indent `SERVICE-NAME` a couple of spaces so it lines up with `CONFIG-FILE`?